packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -680,7 +680,12 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
      * @returns
      */
     $executeRaw(query: TemplateStringsArray | sqlTemplateTag.Sql, ...values: any[]) {
-      return createPrismaPromise((txId, lock, otelCtx) => {
+      return createPrismaPromise(async (txId, lock, otelCtx): Promise<unknown> => {
+        const isInstanceOfSQL = query.constructor.name === 'Sql'
+        if (isInstanceOfSQL && (query as { sql: string }).sql === '') {
+          return null
+        }
+
         if ((query as TemplateStringsArray).raw || (query as sqlTemplateTag.Sql).sql) {
```

Comment:
```suggestion
        if ((query as TemplateStringsArray).raw !== undefined || (query as sqlTemplateTag.Sql).sql !== undefined) {
```

This should be enough.